tests/installation/src/__tests__/extented/security-systems/convenience/acf2.ts

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
We should maybe fix the typo 'extended' in the directory name next iteration when we have more time